plugins/catalog-backend/src/ingestion/processors/ldap/config.ts

```diff
@@ -260,6 +260,11 @@ export function readLdapConfig(config: Config): LdapProviderConfig[] {
     };
   }
 
+  function formatFilter(filter?: string): string | undefined {
+    // Squash multiline query filters to a single line query
+    return filter?.replace(/\n\s*/g, '')?.trim();
```

Comment:
This won't actually work with your example right? Since the `>` wraps it into a string without spaces. Or otherwise just to be sure, does the LDAP query actually allow spaces to begin with just not the trailing newline? i.e. does the following work without this change?

```yaml
filter: >-
  (|
    (cn=foo)
    (cn=bar)
  )
```

Reply:
I have tried with all the different styles available for multiline strings in YAML https://yaml-multiline.info/ and all of them will throw the same error:

`Error: Processor LdapOrgReaderProcessor threw an error while reading location ldap-org:ldaps://localhost, Error: LDAP search at DC=bar,DC=foo failed, missing paren` 

However, it was actually not due to the newlines, but due to the spaces between the parentheses. So I have updated the regex to instead make sure that we remove the extra spaces between each section.